mlflow/models/evaluation/base.py

```diff
@@ -1357,15 +1397,22 @@ def evaluate(
                         def fn(model_input):
                             return model.predict(model_input)
 
+        If omitted, it indicates a static dataset will be used for evaluation instead of a model.
+        In this case, the ``data`` argument must be a Pandas DataFrame that contains model outputs,
+        and the ``predictions`` argument must be the name of the column in ``data`` that contains
+        model outputs.
```

Comment:
This isn't entirely accurate, right? It could be an `mlflow.data.PandasDataset` could also be specified?

Reply:
Updated